starter.sh

```
@@ -0,0 +1,759 @@
+                        echo -e "\033[32m已终止进程 PID=$pid\033[0m"
+                    else
+                        echo -e "\033[33m跳过终止进程\033[0m"
+                    fi
+                fi
+            done
+            sleep 0.5
+        else
+            echo -e "\033[32m端口 $port 未被占用\033[0m"
+        fi
+    elif command -v netstat &> /dev/null; then
+        local connections=$(netstat -tln 2>/dev/null | grep ":$port ")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)